fixtures/cameo/flat-par-can-tri-7x-3w-ir.json

```diff
@@ -0,0 +1,225 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Flat PAR Can Tri 7x 3W IR",
+  "shortName": "CLPFLAT1TRI3WIR",
+  "categories": ["Color Changer", "Dimmer", "Strobe"],
+  "meta": {
+    "authors": ["Fabian"],
+    "createDate": "2018-09-22",
+    "lastModifyDate": "2018-09-22"
+  },
+  "links": {
+    "manual": [
+      "https://adamhall.s3.amazonaws.com/media/MARKEN/CAMEO/CLPFLAT1TRI3WIR/CLPFLAT1TRI3WIR_Cameo_Bedienungsanleitung_EN_DE_FR_ES_PL_IT.pdf"
+    ],
+    "productPage": [
+      "https://www.cameolight.com/en/products/led-flat-par/flat-par-tri-3w-ir-7-x-3-w-high-power-tri-colour-flat-led-rgb-par-light-in-black-housing"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=mKLHiZFlf4Y"
+    ]
+  },
+  "physical": {
+    "dimensions": [180, 175, 115],
+    "weight": 1.2,
+    "power": 25,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "7x 3W RGB LEDs"
+    },
+    "lens": {
+      "degreesMinMax": [30, 30]
+    }
+  },
+  "availableChannels": {
+    "Master Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Color Macro": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 36],
+          "type": "ColorPreset",
+          "comment": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "dmxRange": [37, 73],
+          "type": "ColorPreset",
+          "comment": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "dmxRange": [74, 110],
+          "type": "ColorPreset",
+          "comment": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "dmxRange": [111, 147],
+          "type": "ColorPreset",
+          "comment": "Yellow",
+          "colors": ["#ffff00"]
+        },
+        {
+          "dmxRange": [148, 184],
+          "type": "ColorPreset",
+          "comment": "Magenta",
+          "colors": ["#ff00ff"]
+        },
+        {
+          "dmxRange": [185, 221],
+          "type": "ColorPreset",
+          "comment": "Cyan",
+          "colors": ["#00ffff"]
+        },
+        {
+          "dmxRange": [222, 255],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#ffffff"]
+        }
+      ]
+    },
+    "Strobe": {
+      "capability": {
```

Comment:
At which DMX values is strobe disabled? That should be a separate capability. @cmdrf Could you please check that for us?